src/popup/utils/helper.js

```diff
@@ -254,8 +166,6 @@ export const setBalanceLocalStorage = (balance) => {
 export const getBalanceLocalStorage = () =>
   localStorage.rxjs ? JSON.parse(localStorage.rxjs).balance : '0';
 
-export const getAeppUrl = (v) => new URL(v.connection.port.sender.url);
```

Comment:
Isn't this used anywhere?

Reply:
it is not used, as far CI successfully builds the app